src/components/Button/InternalButton.tsx

```diff
@@ -0,0 +1,60 @@
+import { ElementType } from "react";
+import { classNames, variantClassNames } from "../../util/classes";
+import { ButtonProps } from "./Button";
+
+import { Button } from "./Button";
+
+const DEFAULT_BUTTON_TAG = "button" as const;
+
+export type InternalButtonProps<
+  T extends ElementType = typeof DEFAULT_BUTTON_TAG,
+> = Omit<ButtonProps<T>, "variant" | "size"> & InternalButtonOwnProps;
+
+export interface InternalButtonOwnProps {
+  variant: "wallet" | "wallet-inside";
+}
+
+export const InternalButton = <
```

Comment:
This button can be used to create variants to be used internally to the project, not exposed to the end users.